lib/ffmpeg/qsv/encoder.py

```diff
@@ -92,6 +93,7 @@ def check_metrics(self):
       vars(self).get("lowpower", 0),
       vars(self).get("ldb", 0),
       vars(self).get("strict", 0) == -1,
+      vars(self).get("seek", 0)
```

Comment:
Hmm... why does `tcbrc` strictly require a seek option?  I mean, we should be able to do `tcbrc` without `seek`, right?

Reply:
Yes, but because we lack seek-related test cases, I added this case (Recode_peru_8K_HDR), I think all modes of this case (e.g. CBR/VBR/CQP) need to be tested for seek, and currently the mode of tcbrc is submitted first.